opencti-platform/opencti-front/src/private/components/events/Incidents.js

```diff
@@ -67,6 +64,81 @@ class Incidents extends Component {
     this.setState({ openExports: !this.state.openExports });
   }
 
+  handleToggleSelectEntity(entity, event, forceRemove = []) {
```

Comment:
Depending on the duration, this component should be migrated to use the useEntityToggle hook.
Or maybe, in the firstTime, just mutualize this code to facilitate future migration.

What do you think ?

Reply:
yes good idea! what do you mean by 'mutualize' ? :)

Reply:
I mean, put the logic in a file to use it, like the useEntityToggle file but in the old way.

Reply:
ah yes ok I see !!

Reply:
waiting for the Incident refacto of Marie (we'll just have to add the useEntityToggle hook after the migration

Reply:
So we waiting the merge of the Marie's PR
Good for me